### DISCUSSION TITLE
Dependency not found in Maven Project

### DISCUSSION THREAD

**Discussion by bellaabdelouahab** [bellaabdelouahab/Hotel-Management-System#18] [Q&A] [answered]
I'm trying to add this dependency in my Maven project but it gives me an error like:

"Dependency not found" even though I added the dependency by searching in Intellij IDEA

**Comment by Khalid1G** [comment, marked as answer]
Most likely you just need to reload project in order to force IntelliJ to download the dependency. Right mouse click on the pom.xml in the Project tree -> Maven -> Reload project. I hope it will help.
